app/components/match-tiles/index.tsx

```diff
@@ -1,19 +1,34 @@
 import { Suspense } from 'react';
-import MatchTile from './match-tile';
+import dynamic from 'next/dynamic';
 import Loading from './loading';
 import { MatchResponse } from '@/interfaces/match.interface';
 
+const MatchTile = dynamic(() => import('./match-tile'), { ssr: false });
+
 interface MatchTilesProps {
   matches?: MatchResponse[];
+  expandedMap: Record<number, boolean>;
```

Comment:
used `Map` instead of `Record` in https://github.com/betterbettor/bet-better-react/pull/14/commits/f7cfe696949dd2a11fe3493a40cea08d4659bdee#diff-7f13feabbd133c11ad52754637ba9af88b60a5be1cb39f62ae8742f4bb934491R1